ufazien-cli-py/src/ufazien/cli.py

```
@@ -116,10 +127,20 @@ def create(
     subdomain: Optional[str] = typer.Option(None, "--subdomain", "-s", help="Subdomain"),
     website_type: Optional[str] = typer.Option(None, "--type", "-t", help="Website type (static, php, or build)"),
     database: bool = typer.Option(False, "--database", "-d", help="Create database (PHP only)"),
+    description: Optional[str] = typer.Option(None, "--description", "-D", help="Website description"),
```

Ruff (0.16.1)

[error] Add `from __future__ import annotations` to simplify `typing.Optional`

Add `from __future__ import annotations`

(FA100)